src/toPlaylists.js

```diff
@@ -10,6 +10,12 @@ export const generateSegments = ({ attributes, segmentInfo }) => {
   if (segmentInfo.template) {
     segmentsFn = segmentsFromTemplate;
     segmentAttributes = merge(attributes, segmentInfo.template);
+    // TODO Decide if SegmentList be chosen ahead of SegmentBase.
```

Comment:
```suggestion
    // TODO: Decide if SegmentList be chosen ahead of SegmentBase.
```
So that it shows up in the linter